src/analysis/special_functions/pow.lean

```diff
@@ -1456,6 +1507,28 @@ end
 lemma to_real_rpow (x : ennreal) (z : ℝ) : (x.to_real) ^ z = (x ^ z).to_real :=
 by rw [ennreal.to_real, ennreal.to_real, ←nnreal.coe_rpow, ennreal.to_nnreal_rpow]
 
+lemma injective_rpow_const_of_ne_zero {x : ℝ} (hx : x ≠ 0) :
+  function.injective (λ y : ennreal, y^x) :=
```

Comment:
These functions can be written as `(^ x) : ennreal \to ennreal` if you like.

Reply:
I cannot find the thread either. I'd rather leave them as they are, as I find it more readable that way.

Reply:
Usually the word `injective` goes last. I'd argue this should be called `rpow_left_injective` to match

https://github.com/leanprover-community/mathlib/blob/b537cc08ba7dc490c40e48145ab8a63d12941462/src/data/nat/basic.lean#L1172-L1173